apify-docs-theme/src/theme/NavbarItem/NavbarNavLink.jsx

```diff
@@ -83,7 +83,8 @@ export default function NavbarNavLink({
                 // eslint-disable-next-line no-shadow
                 isActive: (_match, location) => (activeBaseRegex
                     ? isRegexpStringMatch(activeBaseRegex, location.pathname) || dropDownHasActiveItem
-                    : location.pathname.startsWith(activeBaseUrl)),
+                    : location.pathname.startsWith(`/${activeBasePath}`)),
+                activeClassName: 'navbar__link--active',
```

Comment:
this case was never used before. It caused a bug where a navbar element that has a dropdown AND is clickable itself, would not be styled as "active"